Dockerfile

```diff
@@ -20,7 +20,9 @@ RUN addgroup -S -g $EGO_UI_GID $EGO_UI_USER \
 # defaults
 ENV REACT_APP_EGO_CLIENT_ID=ego
 ENV REACT_APP_API=http://localhost:8081/
-ENV PUBLIC_URL="/"
+ARG REACT_APP_PUBLIC_URL="/"
```

Comment:
what is the need for an ARG ? these are for docker build command

Reply:
i had the same setup in dms-ui, i thought that was needed. But can change it back if not!